src/scss/themes/_live.scss

```diff
@@ -1,30 +1,42 @@
 @mixin oTeaserThemeLive {
-    .o-teaser--small.o-teaser--live .o-teaser__content {
-        padding: 12px;
+    // all
+    // > create white text red background
+    .o-teaser--small.o-teaser--live,
+    .o-teaser--hero.o-teaser--live .o-teaser__content,
+    .o-teaser--hero.o-teaser--live .o-teaser__image-container,
+    .o-teaser--large.o-teaser--live .o-teaser__content,
+    .o-teaser--top-story.o-teaser--landscape.o-teaser--live {
+        @include oTeaserInverse;
+        background: oColorsGetPaletteColor('crimson');
+        border: none;
     }
 
-    .o-teaser--small.o-teaser--live.o-teaser--has-image {
-        padding: 12px;
-        .o-teaser__content {
-            padding: 0px;
-        }
+    .o-teaser__content, .o-teaser__meta {
+        color: white;
     }
 
-    .o-teaser--small.o-teaser--live {
-        // create white text red background
-        @include oTeaserInverse;
-        background: oColorsGetPaletteColor('crimson');
-        .o-teaser__content * {
-            color: white;
-            &:hover {
-                color: white;
-            }
+    .o-teaser__tag:hover {
+        color: oColorsMix(oColorsGetPaletteColor('crimson'), oColorsGetPaletteColor('white'), 30) !important;
```

Comment:
anyone can help me to get rid of this `important` ?

Reply:
This is what's causing the issue I think https://github.com/Financial-Times/o-teaser/blob/small-live/src/scss/themes/_standard.scss#L3-L8
I monkey patched it by targeting elements specifically but I'm still looking for a better solution https://github.com/Financial-Times/o-teaser/pull/76/commits/8065026eb99504ae72f1383dd21c3a2d2f804bd7